src/components/WwwFrame/DisclaimersContentful.vue

```diff
@@ -59,7 +59,7 @@ export default {
 		preFetch: true,
 		result({ data }) {
 			// Hide ALL banners on these pages
-			if (isExcludedUrl(globalBannerDenyList, this.$route.path)) return false;
+			if (isExcludedUrl(globalBannerDenyList, [], this.$route.path)) return false;
```

Comment:
To match the cps behavior, this line should be dropped and the `globalBannerDenyList` should be concatenated with the hiddenUrls field in the calls to `isExcludedUrl` below, like it is here https://github.com/kiva/cms-page-server/blob/50cc7ccab8a353516eccb795f9492327bcc101a0/composables/usePromoBanner.ts#L53